### PR TITLE
Redirect to Test Tube

### PR DESCRIPTION
**Pull request by openjck**
Rob Miller gave me the 👍 to do this.

### CODE REVIEW

**Comment by openjck** [comment]
It would be easier to browse the repo at this commit, which currently looks like this:

https://github.com/mozilla/experiments-viewer/tree/762d29837553e7e3179d76d4593b9a60c08e73a4

File: server.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+
+const port = process.env.PORT || 3000;
+
+const server = http.createServer((req, res) => {
+    res.writeHead(301, {Location: 'https://firefox-test-tube.herokuapp.com/'});
+    res.end();
+});
+
+server.listen(port, () => console.log(`Listening on port ${port}...`));
```
Comment (robhudson) [inline server.js]: I wonder if we can also redirect if the user came in to a specific experiment and send them to the right one?

Comment (openjck) [reply server.js]: That's a good point. Dammit, Rob. 😆

**Comment by openjck** [comment]
Updated. :smiley:

**Comment by spasovski** [comment]
r+ though I'll miss this project

**Pull request merged**